src/MoBi.Core/Helper/FormulaDecoder.cs

```diff
@@ -29,7 +29,7 @@ public IFormula GetFormula(object formulaOwner)
    /// <typeparam name="T">The type that has a property of type IFormula that should be accessed</typeparam>
    public abstract class FormulaDecoder<T> : FormulaDecoder
    {
-      public Func<T, IFormula> GetFormula { get; protected set; }
+      public new Func<T, IFormula> GetFormula { get; protected set; }
```

Comment:
I'm going to pull the branch and look at this one. I added this warning quite a while ago and did not notice.

Reply:
ok, I rearranged this `FormulaDecoder` stuff